### ISSUE TITLE
Please add install/usage guide

### ISSUE THREAD

**Issue by kristianmandrup** [fundon/vue-admin#66]
I'm new to Vue and I can't figure out how to use this component... Please advice.

**Comment by niallobrien** [comment]
An installation section within the Readme would be useful.

**Comment by GoldenIron** [comment]
@kristianmandrup What worked for me was: `npm install` after that you can run `npm run dev`. 

If you get an error related to vue-template-compiler mismatch, you need to run this command: `npm install vue-template-compiler@2.0.5`

**Comment by kristianmandrup** [comment]
yes, that is if I want to run it locally for development to add features etc. How do I include it in a project? What are config options/API etc!?

I see there is no `main` entry in the `package.json` so I guess this library is just a more convenient way to include all the bulma etc. dependencies, then include from them?

Then I go to the `client` folder and discover what looks like an app, so I assume I can find a hint there, ie. that it is an example/demo app?
However so far I've only found `@import '~bulma';` in the `App.vue`

Please give better usage instructions in the Readme. We really shouldn't have to guess or find the needle in a haystack to use such a library (or list of components?). Thanks.

**Comment by GoldenIron** [comment]
Yeah a guide "Installation"/"Getting started" would be good. I think there was a request to open a wiki in #78, that also could help here.

**Comment by niallobrien** [comment]
I think simple instructions within the Readme would more than suffice for now.
